Software/scan_key/ErrorSQL.py

```diff
@@ -3,11 +3,19 @@
 import pymysql
 import os
 import smtplib
+import configparser
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
 
 def errorSQL(id, errMessage):
-    cnx = pymysql.connect(user='CATS', password='CATS', host='192.168.1.2', database='CATS', autocommit=True)
+	config = configparser.RawConfigParser() #instantiate config reader
```

Comment:
Long term this file should be made a class and these variables should be pulled into __init__().